src/convertToObject.js

```diff
@@ -6,7 +6,20 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const result = sourceString.split(';').map((el) => el.split(':'));
```

Comment:
[CODE STYLE] - The variable name `result` is too generic and does not convey the content it holds. Consider using a more descriptive name like `styleDeclarations` to enhance readability.